src/XenobladeChroniclesX/Enhancements/ansio/rules.txt

```diff
@@ -0,0 +1,108 @@
+[Definition]
+titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
+name = Anisotropic Filtering
+path = "Xenoblade Chronicles X/Enhancements/Anisotropic Filtering"
+description = Enables anisotropic filtering for ground, wall, and vegetation textures.|Has a minimal performance impact. Made by Cremitif.
```

Comment:
```suggestion
description = Enables anisotropic filtering for ground, wall and vegetation textures to make them appear clearer when viewed from an angle.|Has a minimal performance impact.
```
I can't remember creating this graphic pack... but also some minor spelling corrections.